### PR TITLE
Bookstore: Styling

### PR DESCRIPTION
**Pull request by smunozmo**
# Documentation

All the documentation is on the README file.

## Updates:

*  Add styling using Bootstrap and CSS.

## Notes:

✅  - Used  React.
✅  - Used  Redux.
✅  - Used  React router.
✅  - No linters errors.

### CODE REVIEW

File: src/components/booksdisplay.js [hunk 1]
```diff
@@ -78,19 +80,19 @@ const Books = () => {
...
             <div className="container">
-              <div className="row">
-                <div className="col">
+              <div className="row align-items-center">
+                <div className="col-3 mainTitle">
                   Bookstore CMS
                 </div>
-                <div className="col">
+                <div className="col-2 text-center">
                   <Link className="links" to="/">BOOKS</Link>
```
Comment (hemantso) [inline src/components/booksdisplay.js]: - [x] Add book component should be inside the main container. Kindly check the design and fix it.

- [x] As per design there should be some shadow to main container. Kindly try to fix it.

- [x] As per given design the background of main container is greyish in color. Kindly try to change the color as per design.

Your
image

Original
image

File: src/components/booksdisplay.js [hunk 2]
```diff
@@ -78,19 +80,19 @@ const Books = () => {
   );
 
   return (
-    <div className="container align-middle p-5">
+    <div className="container align-middle p-1">
       <div className="container my-5 align-middle" id="formContainer">
         <div className="card">
           <div className="card-header">
             <div className="container">
-              <div className="row">
```
Comment (hemantso) [inline src/components/booksdisplay.js]: - [x] The card header background color is white as per given design. Let's try to fix it as per the design provided.

File: src/components/booksdisplay.js [hunk 3]
```diff
@@ -35,11 +35,11 @@ const Books = () => {
               <div className="container">
                 <div className="row">
                   <div className="col">
-                    <p className="fs-6 text-secondary">{book.genre}</p>
-                    <p className="fs-5 fw-bold">{book.title}</p>
-                    <p className="fs-5 text-primary">{book.author}</p>
+                    <p className="text-secondary bookGenre">{book.genre}</p>
+                    <p className="fw-bold bookTitle">{book.title}</p>
```
Comment (hemantso) [inline src/components/booksdisplay.js:40]: - [x] As per the provided design, there should be some `margin` between each book card. Kindly check the screenshot in the above comment and Let's try to fix this.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @smunozmo, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- Try to add a `live link` in your README file. 

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by smunozmo** [comment]
Thank you! @hemant-soni-vst-au4 @arslanbisharat

**Pull request merged**